src/model/uniprotkb/sections/SequenceConverter.ts

```diff
@@ -142,6 +142,28 @@ export const convertSequence = (data: UniProtkbAPIModel) => {
       return sequenceFeatures.includes(feature.type);
     });
     sequenceData.featuresData = features;
+    // Add VAR_SEQ to corresponding isoforms
+    if (features && sequenceData.alternativeProducts) {
+      const varSeqs = features.filter(
+        feature => feature.type === FeatureType.VAR_SEQ
+      );
+      sequenceData.alternativeProducts.isoforms = sequenceData.alternativeProducts.isoforms.map(
```

Comment:
Could check that `varSeqs > 0` before L150

Reply:
Good point